content/8-1-community-feedback.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 8.1 Druid Community Feedback - An Open Letter 
+date: '2018-10-14'
+published: true
+authors: ['The Dreamgrove']
+showtoc: true
+description: "An open letter to the community about the state of Druids in patch 8.1."
+---
+
+*This is a supplementary post to the [8.1 Druid Megathread](#) on Reddit. You can follow the discussion there.*
+
+# Balance
+
+**Authors**: Nick, Slippykins, Tettles \
+*[Reddit Thread](#)*
+
+**Balance** is in a decent state with decent DPS across all categories, but no considerable strengths. The {{< spell 272871 "Streaking Stars" >}} nerf hit ST DPS hard. Lively Spirit mechanically feels odd: it provides delayed burst DPS and inefficient use of a healing CD. Additionally, it places a contingency on someone else to maximise your own damage. Bottom line: **{{< spell 279647 "Lively Spirit" >}} does not feel good** to play, but it’s strength relative to other traits makes players incentivised to take it.
+
+**Talent builds** have never varied so little. Most content in the game favours burst DPS over sustained: content where you have breaks between DPS (ex: Mythic+, World Quests), content with periodic damage modifiers (ex: Raid), and content where priority targets must be focused down (ex: Mythic+, Raid). Thus, we have a **cookie-cutter talent build** of {{< spell 205636 "Force of Nature" >}}/{{< spell 102560 "Incarnation: Chosen of Elune" >}}/{{< spell 279620 "Twin Moons" >}}/{{< spell 202770 "Fury of Elune" >}} that emphasises burst DPS at the expense of sustained DPS. Not only that, but the clear damage difference between talents on specific rows (ex: {{< spell 279620 "Twin Moons" >}} vs. {{< spell 202354 "Stellar Drift" >}}/{{< spell 202347 "Stellar Flare" >}}) is just too big.
+
+There is very little **synergy between traits and talents**, and this seems to be a design choice (ex: the hidden 2/3 modifier on {{< spell 272871 "Streaking Stars" >}} with {{< spell 102560 "Incarnation: Chosen of Elune" >}}). This makes it feel like two disjoint systems where one system (talents) is predetermined and the other (traits) is completely at the mercy of Raid/Mythic+ Weekly Chest RNG. In aggregate, this makes us feel like we have **almost no control over our character**.
```

Comment:
While the lack of synergy is boring, it helps with the balance. For instance, Lively Spirit favouring Incarnation pretty much makes it mandatory, where as Streaking Stars having a modifier on it allows Starlord to stay competitive. If there was heavy synergy, we'd not only be locked into using whatever traits we get from weekly chest, we'd also be locked into the talents that synergize with them. Would be worse imho.

Reply:
Agreed that dropping synergies helps with balance. I think there's a healthy way to offer synergies that doesn't detrimentally affect choices by the player -- for ex, the first trait X boosts trait Y, but stacking trait X doesn't boost trait Y past the first. Players could then be seeking combinations of traits that work well together, similar to how we take talents that work well together.

Synergies between traits and talents can then work to disassemble the cookie-cutter build we currently use -- imagine that Streaking Stars had strong synergies with Starlord, enough to incentivise you to take Starlord over Incarnation for ST. Becauseof the existing issues with our talents, we can partially address it by having strategic synergies between traits and talents. I don't think this limits the choice of the player, but rather opens it up in this circumstance. Rather than seeing it as "I must always pair StrS and Starlord together", it's more of a "this trait allows me to take a different talent from my cookie-cutter build."